test/template/www/js/scenarioRestart.js

```diff
@@ -36,7 +36,8 @@ var app = {
         window.codePush.restartApplication(
             function () {
                 /* success */
-                app.sendTestMessage("RESTART_SUCCEEDED");
+                // does not always send...rely on RESTART_FAILED to provide failure state instead of checking for RESTART_SUCCEEDED
+                // app.sendTestMessage("RESTART_SUCCEEDED");
```

Comment:
Interesting...the thing is, if that error was the problem, it would be throwing in every test case, since they're all synchronous. I think this is just an issue with wkwebview. I'm just going to merge it as-is--I've seen all of the platforms not send lifecycle test messages (APPLICATION_RESUMED) at some point, which is why I've removed all of them from the tests and this is just the last one to go lol